src/style/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
please add styles for different modifiers
![image](https://user-images.githubusercontent.com/22824431/224704529-c16a90db-59ff-4d68-b606-2697255d599e.png)